src/App/Fossa/FossaAPIV1.hs

```diff
@@ -69,10 +69,24 @@ import Network.HTTP.Req.Extra (httpConfigRetryTimeouts)
 import Network.HTTP.Types qualified as HTTP
 import Path (File, Path, Rel)
 import Srclib.Types
-import Text.URI (URI)
 import Text.URI qualified as URI
 import Prelude
 
+-- | Represents error emitted via FOSSA instance.
```

Comment:
We're probably going to need to refactor a lot of these types out of this file since it's starting to get pretty large. I don't think we need to do that now, but I'm just noticing that this file is now around 800 lines and holds most of these types and their `FromJSON` definitions.

Reply:
Yeah, this is definitely the biggest target on my personal agenda for a module split.  I don't think it's unreasonable for each API and it's types, instances, and functions to live together in a small module.  We have the same split between entrypoints in the entrypoint refactor

Reply:
Agreed, but I prefer not to do it in this PR's scope. 

I've added techdebt ticket in to review for now - https://github.com/fossas/team-analysis/issues/917